src/sdc11073/mdib/__init__.py

```diff
@@ -1,4 +1,159 @@
+    'AbstractDeviceComponentDescriptorContainer',
+    'AbstractComplexDeviceComponentDescriptorContainer',
+    'MdsDescriptorContainer',
+    'VmdDescriptorContainer',
+    'ChannelDescriptorContainer',
+    'ClockDescriptorContainer',
+    'BatteryDescriptorContainer',
+    'ScoDescriptorContainer',
+    'AbstractMetricDescriptorContainer',
+    'NumericMetricDescriptorContainer',
+    'StringMetricDescriptorContainer',
+    'EnumStringMetricDescriptorContainer',
+    'RealTimeSampleArrayMetricDescriptorContainer',
+    'DistributionSampleArrayMetricDescriptorContainer',
+    'AbstractOperationDescriptorContainer',
+    'SetValueOperationDescriptorContainer',
+    'SetStringOperationDescriptorContainer',
+    'AbstractSetStateOperationDescriptorContainer',
+    'SetContextStateOperationDescriptorContainer',
+    'SetMetricStateOperationDescriptorContainer',
+    'SetComponentStateOperationDescriptorContainer',
+    'SetAlertStateOperationDescriptorContainer',
+    'ActivateOperationDescriptorContainer',
+    'AbstractAlertDescriptorContainer',
+    'AlertSystemDescriptorContainer',
+    'AlertConditionDescriptorContainer',
+    'LimitAlertConditionDescriptorContainer',
+    'AlertSignalDescriptorContainer',
+    'SystemContextDescriptorContainer',
+    'AbstractContextDescriptorContainer',
+    'PatientContextDescriptorContainer',
+    'LocationContextDescriptorContainer',
+    'WorkflowContextDescriptorContainer',
+    'OperatorContextDescriptorContainer',
+    'MeansContextDescriptorContainer',
+    'EnsembleContextDescriptorContainer',
+    'AbstractStateContainer',
+    'AbstractDeviceComponentStateContainer',
+    'AbstractComplexDeviceComponentStateContainer',
+    'AllowedValuesType',
+    'MdsStateContainer',
+    'VmdStateContainer',
+    'ChannelStateContainer',
+    'ClockStateContainer',
+    'BatteryStateContainer',
+    'ScoStateContainer',
+    'AbstractMetricStateContainer',
+    'NumericMetricStateContainer',
+    'StringMetricStateContainer',
+    'EnumStringMetricStateContainer',
+    'RealTimeSampleArrayMetricStateContainer',
+    'DistributionSampleArrayMetricStateContainer',
+    'AbstractOperationStateContainer',
+    'SetValueOperationStateContainer',
+    'SetStringOperationStateContainer',
+    'SetContextStateOperationStateContainer',
+    'SetMetricStateOperationStateContainer',
+    'SetComponentStateOperationStateContainer',
+    'SetAlertStateOperationStateContainer',
+    'ActivateOperationStateContainer',
+    'AbstractAlertStateContainer',
+    'AlertSystemStateContainer',
+    'AlertConditionStateContainer',
+    'LimitAlertConditionStateContainer',
+    'AlertSignalStateContainer',
+    'SystemContextStateContainer',
+    'AbstractContextStateContainer',
+    'PatientContextStateContainer',
+    'LocationContextStateContainer',
+    'WorkflowContextStateContainer',
+    'OperatorContextStateContainer',
+    'MeansContextStateContainer',
+    'EnsembleContextStateContainer',
+]
```

Comment:
only import the sub packages as it was, that structure makes sense and can be exposed to the user.

Reply:
done